pytorch_lightning/trainer/trainer.py

```diff
@@ -526,6 +527,8 @@ def train(self):
                 # update LR schedulers
                 self.optimizer_connector.update_learning_rates(interval='epoch')
 
+                self.checkpoint_connector.one_training_epoch_completed = True
```

Comment:
```suggestion
                self.checkpoint_connector._one_training_epoch_completed = True
```
this shall not be public for users, right?

Reply:
It shoudn't, but I don't expect them to go into `checkpoint_connector`